docs/changelog/107415.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 107415
+summary: Fix `DecayFunctions'` `toString`
```

Comment:
Why does this say `toString`? Unless I am overlooking something this only affects `equals`, and `toString` is not affected at all.

(Thanks though for fixing the issue with the `equals` implementations!)